content/panorama/scripts/custom_game/command-detection.ts

```diff
@@ -0,0 +1,28 @@
+/**
+ * Dictionary object for commands to detect.
+ */
+const shouldDetectCommand: { [key: number]: boolean } = {};
+const addedKeybind: { [key: string]: boolean } = {};
```

Comment:
Couldn't you use `Set<number>` and `Set<string>` here?

Reply:
Yes, can change that